File: SchoolAppClient.NG/src/app/Models/update-exam-schedule-standard-vm.ts
import { CreateExamSubjectWithScheduleVM } from "./create-exam-subject-with-schedule-vm";

export class UpdateExamScheduleStandardVM {
  examScheduleStandardId: number = 0;
  examScheduleId !: number;
  standardId !: number;
  examSubjects: CreateExamSubjectWithScheduleVM[] = [];

}

File: SchoolApiService/ViewModels/UpdateExamScheduleStandardVM.cs
namespace SchoolApiService.ViewModels
{
    public class UpdateExamScheduleStandardVM
    {
        public int ExamScheduleStandardId { get; set; }
        public int ExamScheduleId { get; set; }
        public int StandardId { get; set; }
        public IEnumerable<CreateExamSubjectWithScheduleVM>? ExamSubjects { get; set; }
    }
}
